### ISSUE TITLE
Function escape_param should escape enum values

### ISSUE THREAD

**Issue by robert-cody** [mymarilyn/clickhouse-driver#77]
For all types function escape_param adds quotes around passed values, but for enums it's not which is pretty strange and makes it impossible to pass iterables with enum values:
`
... AND `type` IN %(types)s
`
becomes:
`
... AND `type` IN [type1, type2, type3]
`
which leads to error: 
> DB::Exception: Unknown identifier: type1

**Comment by robert-cody** [comment]
I guess it should be changed from:
`return item.value`
to:
`return escape_param(item.value)`

**Comment by xzkostyan** [comment]
Can you provide query example?

Following example works fine:

```python
In [1]: from clickhouse_driver import Client
   ...: from enum import Enum
   ...: 
   ...: class E(Enum):
   ...:     a = 1
   ...:     b = 2
   ...:     
   ...: 
   ...: c = Client('localhost')
   ...: print(c.execute('SELECT %(x)s', {'x': [E.a, E.b]}))
   ...: 
[((1, 2),)]
```

**Comment by xzkostyan** [comment]
Oh. I see. It will not work on following enum:

```python
class E(Enum):
    a = 'a'
    b = 'b'
```

**Comment by robert-cody** [comment]
Yep, it won't work for any enum value which need to be escaped.

**Comment by xzkostyan** [comment]
This is fixed in 0.0.19 release.
